apps/deploy/upload-dapp-chain/src/common/guards/tg-access-accounts.guard.ts

```diff
@@ -0,0 +1,23 @@
+import { Context } from "telegraf";
+import { CanActivate, ExecutionContext, Injectable } from "@nestjs/common";
+import { TelegrafExecutionContext, TelegrafException } from "nestjs-telegraf";
+
+import "dotenv/config";
+
+@Injectable()
+export class TgAccessAccountsGuard implements CanActivate {
+  constructor() {}
+
+  canActivate(context: ExecutionContext): boolean {
+    const ctx = TelegrafExecutionContext.create(context);
+    const { from } = ctx.getContext<Context>();
+
+    const accounts = process.env.TELEGRAM_ACCESS_ACCOUNTS.split(",");
```

Comment:
Let's keep it in database. And pls add functionality to add new account of person who can use this bot